samples/long_running_job/long_running_job.R

```diff
@@ -11,7 +11,7 @@ install_github("azure/doazureparallel")
 library(doAzureParallel)
 
 credentialsFileName <- "credentials.json"
-clusterFileName <- "cluster.json"
+clusterFileName <- "test_cluster.json"
```

Comment:
revert?

Reply:
reverted. updated all occurrence of test_cluster.json to cluster.json